applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -0,0 +1,296 @@
+
+  * ``2`` - Notification that a *publish event* has been received on a topic the client is subscribed to.
+  * ``7`` - Acknowledgment of the subscribe request.
+  * ``8`` - Acknowledgment of the unsubscribe request.
+
+* The ``<result>`` value is an integer and it can assume the following values:
+
+  * ``0`` - Value indicating the acknowledgment of the connection request.
+  * *Negative Value* - Error code indicating the reason for the failure.
+
+* The ``<msg>`` value is a string and it contains the payload on the topic published.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XMQTTSUB="nrf91/slm/mqtt/topic0",1,0
+   OK
+   #XMQTTEVT: 7,0
+
+::
+
+   #XMQTTEVT: 2,0," nrf91/slm/mqtt/topic1","a test message"
+
+::
+
+   AT#XMQTTSUB="nrf91/slm/mqtt/topic1",1,1
+   OK
+   #XMQTTEVT: 7,0
+
+::
+
+   AT#XMQTTSUB=1,2,"nrf91/slm/mqtt/topic2"
+   OK
+   #XMQTTEVT: 7,0
+
+::
+
+   AT#XMQTTSUB="nrf91/slm/mqtt/topic3",0,2
+   OK
+   #XMQTTEVT: 8,0
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+----
+
+MQTT Publish #XMQTTPUB
+======================
+
+The ``#XMQTTPUB`` command allows you to publish messages on MQTT topics.
+
+Set command
+-----------
+
+The set command allows you to publish messages on MQTT topics.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XMQTTPUB=<topic>,<datatype>,<msg>,<qos>,<retain>
+
+
+* The ``<topic>`` parameter is a string and it indicates the topic on which data is published.
+* The ``<datatype>`` parameter can accept one the following values:
+
+  * ``0`` - hexidecimal string (e.g. "DEADBEEF" for 0xDEADBEEF)
+  * ``1`` - plain text (default value)
+  * ``2`` - JSON
+  * ``3`` - HTML
+  * ``4`` - OMA TLV
+
+* The ``<msg>`` parameter is a string and it contains the payload on the topic being published.
+  The max ``NET_IPV4_MTU`` is 576 bytes.
+  No NULL characters can be in the middle.
+* The ``<qos>`` parameter is an integer.
+  It can assume the following values:
+
+  * ``0`` - Almost once
+  * ``1`` - At least once
+  * ``2`` - Exactly once
+
+* The ``<retain>`` parameter is an integer and it indicates the topic on which the data is being published.
+  When ``1``, it indicates that the broker has to store the message persistently.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XMQTTEVT: <evt_type>,<result>
+
+* The ``<evt_type>`` value is an integer and it can assume the following values:
+
+  * ``3`` - Acknowledgment for the published message with QoS 1.
+  * ``4`` - Reception confirmation for the published message with QoS 2.
+  * ``5`` - Release of the published message with QoS 2.
+  * ``6`` - Confirmation to a publish release message with QoS 2.
+
+* The ``<result>`` value is an integer and it can assume the following values:
+
+  * ``0`` - Value indicating the acknowledgment of the connection request.
+  * *Negative Value* - Error code indicating the reason for the failure.
+
+Examples
+~~~~~~~~
+
+::
```

Comment:
@lats1980 Are the following examples updated? They appear to be messing some parameters from the syntax, or the syntax is missing some square parenthesys (making msg, qos, and retain optional)